Sources/LLVMSwift/TargetData.swift

```diff
@@ -168,6 +44,139 @@ public struct TargetData {
   public func sizeOfTypeInBits(_ type: IRType) -> Int {
     return Int(LLVMSizeOfTypeInBits(llvm, type.asLLVM()))
   }
+
+  /// Computes the number of bits necessary to hold a value of the given type
+  /// for this target environment.
+  ///
+  /// - parameter type: The type to compute the size of.
+  ///
+  /// - returns: The size of the type in bits.
+  public func abiSizeOfTypeInBits(_ type: IRType) -> Int {
+    return Int(LLVMSizeOfTypeInBits(llvm, type.asLLVM()))
+  }
+
+  /// The current platform byte order, either big or little endian.
+  public var byteOrder: ByteOrder {
+    return ByteOrder(llvm: LLVMByteOrder(llvm))
+  }
+
+  /// Creates a string representation of the target data.
+  public var layoutString: String {
+    let str = LLVMCopyStringRepOfTargetData(llvm)!
```

Comment:
The implementation (I know, fragile) just calls strdup on a `std::string`'s `c_str()`...

I don't think there's a chance this will be null.